python/examples/searching-for-packages.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python3
+
+# Query all packages that match part of their name with the searched string.
+# A Python equivalent for `rpm -qa kernel*`
+
+import rpm
+
+ts = rpm.TransactionSet()
+mi = ts.dbMatch()
+mi.pattern("name", rpm.RPMMIRE_GLOB, "kernel*")
```

Comment:
I would make the package name in these examples something you pass on the command line to make them actually usable on real systems which are unlikely to have "hello" package installed.